packages/unmock-core/package.json

```diff
@@ -43,5 +46,9 @@
     "uuid": "^3.3.2",
     "winston": "^3.2.1",
     "xregexp": "^4.2.4"
+  },
+  "devDependencies": {
+    "@types/deep-equal": "^1.0.1",
+    "deep-equal": "^1.1.0"
```

Comment:
We've kept each package's devDependencies at the top-level package.json, I don't see any reason to put them here (and it does not work as expected I think).